Set the correct paths for gettext and include the installation directory to the sys.path
Index: wakeonplan-0.0.0+bzr61/bin/wakeonplan
===================================================================
--- wakeonplan-0.0.0+bzr61.orig/bin/wakeonplan	2012-09-28 20:26:31.450766957 +0200
+++ wakeonplan-0.0.0+bzr61/bin/wakeonplan	2012-09-28 20:57:49.986711074 +0200
@@ -18,25 +18,33 @@
 import sys
 import os
 
+import locale
 import gettext
-from gettext import gettext as _
-gettext.textdomain('wakeonplan')
+from locale import gettext as _
+locale.bindtextdomain('wakeonplan', '/usr/share/wakeonplan/share/locale')
+locale.textdomain('wakeonplan')
 
 from argparse import ArgumentParser
 
 # Add project root directory (enable symlink and trunk execution)
 PROJECT_ROOT_DIRECTORY = os.path.abspath(
     os.path.dirname(os.path.dirname(os.path.realpath(sys.argv[0]))))
+print "PROJECT_ROOT_DIRECTORY: %s"%(PROJECT_ROOT_DIRECTORY)
 
 python_path = []
-if os.path.abspath(__file__).startswith('/opt'):
+if os.path.abspath(__file__).startswith('/usr'):
     syspath = sys.path[:] # copy to avoid infinite loop in pending objects
     for path in syspath:
-        opt_path = path.replace('/usr', '/opt/extras.ubuntu.com/wakeonplan')
+        opt_path = path.replace('/usr', '/usr/share/wakeonplan')
+        print "Adding: %s"%(opt_path)
         python_path.insert(0, opt_path)
         sys.path.insert(0, opt_path)
+    sys.path.insert(0, "/usr/share/wakeonplan")
+    sys.path.insert(0, "/usr/share/wakeonplan_lib")
+    os.putenv("XDG_DATA_DIRS", "%s:%s" % ("/usr/share/wakeonplan/share/", os.getenv("XDG_DATA_DIRS", "")))
 if (os.path.exists(os.path.join(PROJECT_ROOT_DIRECTORY, 'wakeonplan'))
     and PROJECT_ROOT_DIRECTORY not in sys.path):
+    print "Adding PROJECT_ROOT_DIRECTORY: %s"%(PROJECT_ROOT_DIRECTORY)
     python_path.insert(0, PROJECT_ROOT_DIRECTORY)
     sys.path.insert(0, PROJECT_ROOT_DIRECTORY)
 if python_path:
Index: wakeonplan-0.0.0+bzr61/wakeonplan/AboutWakeonplanDialog.py
===================================================================
--- wakeonplan-0.0.0+bzr61.orig/wakeonplan/AboutWakeonplanDialog.py	2012-09-28 20:26:31.434766956 +0200
+++ wakeonplan-0.0.0+bzr61/wakeonplan/AboutWakeonplanDialog.py	2012-09-28 20:50:09.242724781 +0200
@@ -14,9 +14,11 @@
 # with this program.  If not, see <http://www.gnu.org/licenses/>.
 ### END LICENSE
 
+import locale
 import gettext
-from gettext import gettext as _
-gettext.textdomain('wakeonplan')
+from locale import gettext as _
+locale.bindtextdomain('wakeonplan', '/usr/share/wakeonplan/share/locale')
+locale.textdomain('wakeonplan')
 
 import logging
 logger = logging.getLogger('wakeonplan')
Index: wakeonplan-0.0.0+bzr61/wakeonplan/PreferencesWakeonplanDialog.py
===================================================================
--- wakeonplan-0.0.0+bzr61.orig/wakeonplan/PreferencesWakeonplanDialog.py	2012-09-28 20:26:31.398766958 +0200
+++ wakeonplan-0.0.0+bzr61/wakeonplan/PreferencesWakeonplanDialog.py	2012-09-28 20:50:22.610724382 +0200
@@ -22,9 +22,11 @@
 
 from gi.repository import Gio as gio # pylint: disable=E0611
 
+import locale
 import gettext
-from gettext import gettext as _
-gettext.textdomain('wakeonplan')
+from locale import gettext as _
+locale.bindtextdomain('wakeonplan', '/usr/share/wakeonplan/share/locale')
+locale.textdomain('wakeonplan')
 
 import logging
 logger = logging.getLogger('wakeonplan')
Index: wakeonplan-0.0.0+bzr61/wakeonplan/WakeonplanWindow.py
===================================================================
--- wakeonplan-0.0.0+bzr61.orig/wakeonplan/WakeonplanWindow.py	2012-09-28 20:26:31.410766958 +0200
+++ wakeonplan-0.0.0+bzr61/wakeonplan/WakeonplanWindow.py	2012-09-28 20:50:38.342723915 +0200
@@ -22,10 +22,12 @@
 from gi.repository import Pango as pango
 from gi.repository import Notify
 
+import locale
 import gettext
-from gettext import gettext as _
+from locale import gettext as _
 from gettext import ngettext
-gettext.textdomain('wakeonplan')
+locale.bindtextdomain('wakeonplan', '/usr/share/wakeonplan/share/locale')
+locale.textdomain('wakeonplan')
 
 from gi.repository import Gtk as gtk # pylint: disable=E0611
 from gi.repository import Gio as gio # pylint: disable=E0611
Index: wakeonplan-0.0.0+bzr61/wakeonplan/__init__.py
===================================================================
--- wakeonplan-0.0.0+bzr61.orig/wakeonplan/__init__.py	2012-09-28 20:26:31.422766957 +0200
+++ wakeonplan-0.0.0+bzr61/wakeonplan/__init__.py	2012-09-28 20:48:52.882727053 +0200
@@ -16,9 +16,11 @@
 
 import optparse
 
+import locale
 import gettext
-from gettext import gettext as _
-gettext.textdomain('wakeonplan')
+from locale import gettext as _
+locale.bindtextdomain('wakeonplan', '/usr/share/wakeonplan/share/locale')
+locale.textdomain('wakeonplan')
 
 from gi.repository import Gtk # pylint: disable=E0611
 
Index: wakeonplan-0.0.0+bzr61/wakeonplan_lib/helpers.py
===================================================================
--- wakeonplan-0.0.0+bzr61.orig/wakeonplan_lib/helpers.py	2012-09-28 20:26:31.382766959 +0200
+++ wakeonplan-0.0.0+bzr61/wakeonplan_lib/helpers.py	2012-09-28 20:49:08.762726580 +0200
@@ -21,9 +21,11 @@
 from . wakeonplanconfig import get_data_file
 from . Builder import Builder
 
+import locale
 import gettext
-from gettext import gettext as _
-gettext.textdomain('wakeonplan')
+from locale import gettext as _
+locale.bindtextdomain('wakeonplan', '/usr/share/wakeonplan/share/locale')
+locale.textdomain('wakeonplan')
 
 def get_builder(builder_file_name):
     """Return a fully-instantiated Gtk.Builder instance from specified ui 
Index: wakeonplan-0.0.0+bzr61/wakeonplan_lib/wakeonplanconfig.py
===================================================================
--- wakeonplan-0.0.0+bzr61.orig/wakeonplan_lib/wakeonplanconfig.py	2012-09-28 20:26:31.370766958 +0200
+++ wakeonplan-0.0.0+bzr61/wakeonplan_lib/wakeonplanconfig.py	2012-09-28 21:14:18.046681683 +0200
@@ -27,15 +27,16 @@
 
 # Where your project will look for your data (for instance, images and ui
 # files). By default, this is ../data, relative your trunk layout
-__wakeonplan_data_directory__ = '../data/'
+__wakeonplan_data_directory__ = '/usr/share/wakeonplan/share/wakeonplan/'
 __license__ = 'GPL-3'
 __version__ = 'VERSION'
 
 import os
-
+import locale
 import gettext
-from gettext import gettext as _
-gettext.textdomain('wakeonplan')
+from locale import gettext as _
+locale.bindtextdomain('wakeonplan', '/usr/share/wakeonplan/share/locale')
+locale.textdomain('wakeonplan')
 
 class project_path_not_found(Exception):
     """Raised when we can't find the project directory."""
